handlers/transcode.go

```diff
@@ -107,11 +100,11 @@ func RunTranscodeProcess(mistClient clients.MistAPIClient, request TranscodeSegm
 		return fmt.Errorf("failed to start MistProcLivepeer: %s", err)
 	}
 
-	dir, _ := url.Parse(".")
-	uploadDir := inputUrl.ResolveReference(dir)
+	dir, _ := url.Parse("transcoded/")
```

Comment:
does it need to be hard-coded or can it be configurable in request payload?

Reply:
At least initially, hardcoded - can look at making it variable in the future if needed